builtin/game/chat.lua

```diff
@@ -738,17 +738,26 @@ core.register_chatcommand("fixlight", {
 })
 
 core.register_chatcommand("mods", {
-	params = "",
-	description = S("List mods installed on the server"),
+	params = S("[-t]"),
+	description = S("List mods installed on the server (-t: output in chat)"),
 	privs = {},
 	func = function(name, param)
-		local mods = core.get_modnames()
-		if #mods == 0 then
-			return true, S("No mods installed.")
-		else
-			return true, table.concat(core.get_modnames(), ", ")
+		if param == "" then
+			if INIT ~= "game" then
```

Comment:
checking `INIT` here is totally wrong, you can easily determine if it's possible to show a formspec using `core.get_player_by_name(name)`

Reply:
Why is the way wrong? Although it works fine without problems. Also, I saw it is used for checking the same thing in `do_help_cmd()` of `/help` command.

Reply:
The help chat command is is in `builtin/common/`.  If you're in `builtin/game/` then `INIT` will always be `game`

Reply:
But if I am in`/builtin/common`? Then which value will it have?

Reply:
INIT will be "game" "client" or "mainmenu" depending on which environment it's called from